flocker/node/functional/test_docker.py

```diff
@@ -324,24 +330,6 @@ def started(response):
         d.addCallback(started)
         return d
 
-    def build_slow_shutdown_image(self):
-        """
-        Create a Docker image that takes a while to shut down.
-
-        This should really use Python instead of shell:
-        https://clusterhq.atlassian.net/browse/FLOC-719
-
-        :return: The name of created Docker image.
-        """
-        path = FilePath(self.mktemp())
-        path.makedirs()
-        path.child(b"Dockerfile.in").setContent("""\
-FROM busybox
-CMD sh -c "trap \"\" 2; sleep 3"
-""")
-        image = DockerImageBuilder(test=self, source_dir=path)
-        return image.build()
-
```

Comment:
Mildly related to issue. This is another invocation of `DockerImageBuilder.build`, although strictly speaking we didn't _have_ to change it because it wasn't used.